test/functional/Java8andUp/playlist.xml

```diff
@@ -2672,5 +2672,27 @@
 			<impl>ibm</impl>
 		</impls>
 	</test>
+	
+	<test>
+		<testCaseName>DupClassNameTest</testCaseName>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) $(JAVA_COMMAND) $(JVM_OPTIONS) -Xshareclasses:name=DupClassNameTest,reset \
+			-cp $(Q)$(LIB_DIR)$(D)asm-all.jar$(P)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)DupClassNameTest$(D)version1$(Q) \
```

Comment:
Do we need `asm-all.jar` for this test?

Reply:
No. Removed. Also removed` RESOURCES_DIR`. 

Seems that `TESTNG` include `testng.jar` and` jcommander.jar`, but in build.xml the class DupClassNameTest is compiled with only `testng.jar`. It is working now. Not sure if it will cause any issue in the future. 
`<pathelement location="${LIB_DIR}/testng.jar" />`